tests/integration/master_slave/spec/default_spec.rb

```diff
@@ -0,0 +1,185 @@
+require "spec_helper"
+require "json"
+
+class ServiceNotReady < StandardError
+end
+
+context "when manual failover" do
+  describe server(:namenode1) do
+    it "should be ready" do
+      response = nil
+      retry_and_sleep do
+        response = fetch "http://#{server(:namenode1).server.address}:5070/jmx"
+      end
+      expect(response.class).to eq(Net::HTTPOK)
+    end
+
+    it "should return NameNodeStatus == active" do
+      # {
```

Comment:
i do not understand what the comment means.
is it for debugging or planning to add more tests of the contents?